### ISSUE TITLE
ensure the examples are up to date

### ISSUE THREAD

**Issue by annastuchlik** [rjeczalik/serverless-examples#10]
This issue is to ensure that all the examples are updated.

- [ ] Java
 - Merge https://github.com/scylladb/serverless-examples/pull/8.
 - Fix https://github.com/scylladb/siren-frontend/issues/3402
 - Contact persons: @avelanarius @Bouncheck 
- [x]   Python - up-to-date as of 8/2 2023
- Contact persons:  @avelanarius @Lorak-mmk @fruch 
- [ ] Rust
- Merge https://github.com/scylladb/serverless-examples/pull/9 + make updates according to the feedback from @wprzytula in that PR.
- Are any updates required regarding TokenAwarePolicy **-NO**
- Contact persons: @piodul @wprzytula @avelanarius 
 - [ ] Go 
 - Update the example
 - Contact person: @avelanarius 
- [ ] C++
- Add the example as is @rjeczalik 
 - Update the example.
 - Contact person: @avelanarius

**Comment by fruch** [comment]
The python example looks o.k.

Which updates does it need ?

**Comment by annastuchlik** [comment]
@fruch Right, it should go "update the example if needed". I just wanted to confirm that the examples are added and up to date. Thanks for your confirmation!

**Comment by wprzytula** [comment]
> Rust
...
Are any updates required regarding https://github.com/scylladb/siren-frontend/issues/3402?

The described issue is irrelevant given current Rust driver design (`DefaultPolicy` encompasses both `DCAwarePolicy`'s and `TokenAwarePolicy`'s roles). If the question only concerns the Rust driver example in this repository, then there's nothing to do about it.
